www/calendar/inner.js

```diff
@@ -818,21 +818,37 @@ define([
             });
         }
         if (APP.$calendars) { APP.$calendars.append(calendar); }
-        return calendar;
+        return $calendar;  // return jQuery element
+    };
+
+    var appendCalendarEntries = function (teamId, filter) {
+        var calendars = filter(teamId);
+        var $entriesContainer = $('<div class="cp-calendar-entries"></div>');
```

Comment:
We should try to use hyperscript (the `h` function) and avoid jQuery as much as possible, especially when we want to make mobile improvements.